tests/test_extraction.py

```diff
@@ -1,31 +1,208 @@
-import unittest
-from textextraction import doc_process_toolkit as dpt
+import os
+from unittest import TestCase, skipUnless, main
+from textextraction.textextraction import (TextExtraction, PDFTextExtraction,
+                                           textextractor)
 
+ALL_INSTALLED = os.getenv("ALL_INSTALLED")
```

Comment:
I really don't like this mechanism for testing. If the requirements aren't installed, I think the tests should fail.